sites/next.skeleton.dev/src/examples/resources/cookbook/image-layouts/ExampleFeatured.astro

```diff
@@ -0,0 +1,28 @@
+<!-- Flex -->
+<section class="flex justify-center">
```

Comment:
The use of `section > div` seems redundant. I'd recommend we keep the `section` as the parent, and move the grid style up to here.